docs/cue/reference/components/sources/eventstoredb_metrics.cue

```diff
@@ -2,7 +2,6 @@ package metadata
 
 components: sources: eventstoredb_metrics: {
 	title: "EventStoreDB Metrics"
-	alias: "eventstoredb"
```

Comment:
The alias is supposed to be used for renaming components, but this one was never released as `eventstoredb`